geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStore.scala

```diff
@@ -170,7 +170,8 @@ class AccumuloDataStore(val connector: Connector,
           sft.setTableSharingPrefix("")
         }
         // noinspection ScalaDeprecation
-        metadata.read(typeName, SimpleFeatureTypes.ENABLED_INDEXES_OLD).foreach(sft.setEnabledTables)
+        metadata.read(typeName, SimpleFeatureTypes.ENABLED_INDEXES_OLD)
+            .foreach(sft.getUserData.put(SimpleFeatureTypes.ENABLED_INDEXES, _))
```

Comment:
Is the metadata for ENABLED_INDEXES_OLD one entry or multiple?

In the case of the latter, this'd clobber things.